webpack.common.js

```diff
@@ -0,0 +1,57 @@
+const path = require("path");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+const public = path.resolve(__dirname, "public");
+
+module.exports = {
+  entry: "./src/index.tsx",
+  output: {
+    filename: "[name].[contenthash].js",
+    path: path.resolve(__dirname, "build"),
+    publicPath: "/",
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.js|\.jsx$/,
+        exclude: /node_modules/,
+        use: {
+          loader: "babel-loader",
+
+          options: {
+            presets: ["@babel/preset-env"],
+          },
+        },
+      },
+
+
+      {
+        test: /\.(tsx|ts)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: "ts-loader",
+        },
+      },
+
+      {
+        test: /\.scss$/,
+        use: [MiniCssExtractPlugin.loader, "css-loader", "sass-loader"],
+      },
+    ],
+  },
+
+  resolve: {
+    extensions: [".*", ".js", ".jsx", ".ts", ".tsx"],
```

Comment:
I think once you've added the `.*` extension then you don't need to specify any ext explicitly.